server/bin/pbench-audit-server.sh

```diff
@@ -772,8 +774,10 @@ if [[ -s ${users_report} ]]; then
     printf "\nend-${eTS}: users hierarchy: $USERS\n" >> ${report}
 fi
 
-if [[ -s ${report} ]]; then
-    log_error "${TS}(${PBENCH_ENV}) - audit found problems, please review ${report}"
+# Move the completed intermediate report to be the latest.
+mv ${report} ${latest_report}
+if [[ -s "${latest_report}" ]]; then
+    log_error "${TS}(${PBENCH_ENV}) - audit found problems, please review ${latest_report}"
 fi
```

Comment:
Modulo my previous comment, shouldn't we be skipping the move if the report is empty?  (Why is it important to save an empty report?  Why wouldn't it better to skip it and retain the previous non-empty report?)  E.g.,
```
# If there is something to report, move the completed intermediate report to be the latest.
if [[ -s "${report}" ]]; then
    mv ${report} ${latest_report}
    log_error "${TS}(${PBENCH_ENV}) - audit found problems, please review ${latest_report}"
fi
```